example/react-native.config.js

```diff
@@ -7,8 +7,24 @@ if (process.argv.includes("--config=metro.config.macos.js")) {
     reactNativePath: "node_modules/react-native-windows",
   };
 } else {
+  const path = require("path");
+  const sourceDir = "android";
   module.exports = {
     project: {
+      android: {
+        sourceDir,
+        manifestPath: path.relative(
```

Comment:
Why do we need to specify the manifest path explicitly?

Reply:
I wonder if there is a backwards incompatible change for midgard.

Reply:
`react-native config` can't find it otherwise because we have a rather unconventional project structure. This is only needed for `run-android` to work. Without it, everything else should work as usual.